src/main/java/io/github/tereshchenkoalex/chapter3/Triangle.java

```diff
@@ -0,0 +1,39 @@
+package io.github.tereshchenkoalex.chapter3;
+
+/**
+ * Chapter 3. Task 1. Page 65
+ */
+public class Triangle {
+
+    double area;
+    int height;
+    int length;
+
+    public static void main(String [] args) {
+
+    int x = 0;
```

Comment:
Why have you declared this variable here? it is recommended to declare a variable as close to its usage, as possible.